app.py

```diff
@@ -94,7 +94,7 @@ def load_adventures_per_level(level, keyword_lang):
                 loaded_programs[program_key] = program
 
     all_adventures = []
-    adventures = ADVENTURES[g.lang].get_adventures(keyword_lang)
+    adventures = ADVENTURES[keyword_lang].get_adventures(keyword_lang)
```

Comment:
This is an incorrect fix and should not have been merged. When retrieving the adventures we want to get the adventures of the user language, not the user keyword language.

We do however want to parse all commands with the keyword language. But this already happens correctly.

So: What is the purpose of this change?